src/Microsoft.AspNet.Razor.Test.Sources/project.json

```diff
@@ -0,0 +1,13 @@
+﻿{
+  "version": "1.0.0-*",
```

Comment:
I thought of that as well, but this isn't specific to the other Razor dependencies. Don't feel strongly about it though. /cc @Eilon